.github/workflows/test.yml

```
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  RUBY_VERSION: 3.1.1
+  RUBY_VERSION: 3.2.2
```

This may fix the CI error:

```suggestion
  RUBY_VERSION: 3.2.2
  DISABLE_SPRING: 1
```